.travis.yml

```diff
@@ -19,10 +19,14 @@ matrix:
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.7.*
-    - php: 5.6
-      env: SYMFONY_VERSION=2.8.*
+      env: DEPENDENCIES="symfony/lts:^2"
+    - php: 7.1
+      env: DEPENDENCIES="symfony/lts:^3"
+    - php: 7.1
+      env: STABILITY="dev"
   allow_failures:
+    - php: 7.1
+      env: STABILITY="dev"
     - php: hhvm
```

Comment:
I suggest removing testing on HHVM as HHVM stopped bothering about PHP compat and Symfony dropped support for it. I see no interest into debugging HHVM failures if they don't care about PHP compat, so I don't care about having it on CI (I removed it from any project I maintain for which I reworked the Travis config after their announcement)